src/etc/inc/util.inc

```diff
@@ -2474,5 +2488,21 @@ function get_smart_drive_list() {
 	sort($disk_list);
 	return $disk_list;
 }
+/* Write the DHCP6 DUID file */
 
+function write_dhcp6_duid($dhcp6duid) {
+	// Create the hex array from the dhcp6duid config entry and write to file
+
+	$temp = str_replace(":","",$dhcp6duid);
+	$duid_binstring = pack("H*",$temp);
+	if ($fd = fopen("/conf/dhcp6c_duid", "wB")){
+		fwrite($fd, $duid_binstring);
+		fclose($fd);
+		return 1;
```

Comment:
Just copying what I'd seen elsewhere, :) 
I'll use true and false where possible in future.